src/common/x64/native_clock.cpp

```diff
@@ -10,25 +10,47 @@
 #include "common/uint128.h"
 #include "common/x64/native_clock.h"
 
+#ifdef _MSC_VER
+#include <intrin.h>
+#endif
+
 namespace Common {
 
+inline u64 FencedRDTSC() {
```

Comment:
```suggestion
static u64 FencedRDTSC() {
```

So this has internal linkage